### PR TITLE
chore: move client from SDK to numaflow

### PR DESCRIPTION
**Pull request by kohlisid**

### CODE REVIEW

File: pkg/sink/interface.go [hunk 1]
```diff
@@ -16,13 +13,6 @@ type Datum interface {
 	ID() string
 }
 
-// Client contains methods to call a gRPC client.
-type Client interface {
```
Comment (jy4096) [inline pkg/sink/interface.go:20]: do we have the Client interface for function? I don't see in the change files...

File: pkg/sink/server/server_test.go [hunk 2]
```diff
@@ -30,74 +26,48 @@ func Test_server_sink(t *testing.T) {
...
+		return result
+	})
+	// note: using actual UDS connection
+	ctx, cancel := context.WithTimeout(context.Background(), 6*time.Second)
+	go func() {
+		time.Sleep(3 * time.Second)
+		cancel()
+	}()
+	err = New().RegisterSinker(sinkHandler).Start(ctx, WithSockAddr(socketFile.Name()), WithServerInfoFilePath(serverInfoFile.Name()))
+	//err := New().RegisterMapper(mapHandler).Start(ctx, WithSockAddr(socketFile.Name()), WithServerInfoFilePath(serverInfoFile.Name()))
```
Comment (jy4096) [inline pkg/sink/server/server_test.go:49]: clean up

File: pkg/sink/interface.go (hunk 1 above)
Comment (kohlisid) [reply pkg/sink/interface.go:20]: Updated

File: pkg/sink/server/server_test.go (hunk 2 above)
Comment (kohlisid) [reply pkg/sink/server/server_test.go]: Done

**Review by jy4096 — APPROVED** [review]
Looks good to me

**Review by yhl25 — APPROVED** [review]

**Pull request merged**